README.md

```diff
@@ -71,10 +77,17 @@ Icons
 The icons should be 16x16 pixels. I recommend the [Fugue icon set](http://p.yusukekamiyamane.com/) (shadowless).
 
 
-kthxbye
 
--[joe](http://joewalnes.com)
+CHANGES FROM ORIGINAL
+-----------------------
 
+- can now specify the tag type that the clickable context-menu element should be.
+Defaults to <span> instead of <a>, since clicks don't necessarily take someone to a new resource/URI
```

Comment:
<a> (or <button>) is more semantically correct than <span>. It symbolizes that this is a clickable action.